### DISCUSSION TITLE
Ability to rename show title

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#9556] [Ideas & Feature requests]
Since I have switched to TMDB as indexer I'm running into some issues that some shows are only stored there by title without any additions.

For example: LEGO Masters (2019) 
https://www.themoviedb.org/tv/88754-lego-masters-au
This is the Australian version, but the episodes will be stored as: `LEGO Masters - S01E01`

This results in searches picking up the US version of this show, I can add Scene exceptions:
![image](https://user-images.githubusercontent.com/4103090/118775504-4b65f180-b887-11eb-8594-a06438340b60.png)

But the original `LEGO Masters` title will still always be searched.
![image](https://user-images.githubusercontent.com/4103090/118775665-78b29f80-b887-11eb-8470-2b30ec4b18a3.png)

**Comment by ddalben** [comment]
I have the same issue with the same show.  TV Maze doesn't differentiate the markets in the show name.  The only distinction is Network or entries in the Tags or aka areas.  See here: [Lego Masters](https://www.tvmaze.com/shows/41989/lego-masters)

There's a couple of ways this could be resolved:
1. Allow Show Name change in Medusa
2. Allow a Scene Exception to be the default search for the show
3. Change the tvmaze show name to reflect the market (not something they do, but I've asked the question).

Until then, these franchised shows will be difficult to manage.

**Reply by p0psicles** [reply]
In the future no. 2 will be available as an option.
Work has already started on that. Just need to finish that.

**Reply by ddalben** [reply]
That's great to hear.  For now I've rolled these shows back to tvdb.

Option 3. was shut down pretty quickly.
